charts/private-link/values.yaml

```diff
@@ -0,0 +1,171 @@
+################################################################################
+# Global options
+################################################################################
+global:
+  image:
+    # global image pull policy to use for all container images in the chart
+    # can be overridden by individual image pullPolicy
+    pullPolicy:
+    # global registry to use for all container images in the chart
+    # can be overridden by individual image registry
+    registry: registry.synadia.io
+
+  # global labels will be applied to all resources deployed by the chart
+  labels: {}
+
+################################################################################
+# Private Link Deployment and associated resources
+################################################################################
+
+############################################################
+# image pull secret
+############################################################
+# when enabled, the registry for images in the private link deployment
+# will default to global.image.registry
+imagePullSecret:
+  enabled: false
+  registry:
+  username:
+  password:
+
+  # merge or patch the image pull secret
+  # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#secret-v1-core
+  merge: {}
+  patch: []
+  # defaults to "{{ include "spl.fullname" $ }}-regcred"
+  name:
+
+############################################################
+# config
+############################################################
+# due to the potentially sensitive nature of some of the configuration
+# options, the entire configuration will be stored in a Kubernetes Secret
+#
+config:
+  # URL for Synadia Control Plane
+  # defaults to Synadia Cloud
+  platformURL:
+  # URL for NATS Server
+  natsURL:
+  # Synadia Cloud or Control Plane Token
+  token:
+  # TLS for connection to NATS Server
+  tls:
+    # set secretName in order to mount an existing secret to dir
+    secretName:
+    dir: /etc/private-link/certs
+    cert: tls.crt
+    key: tls.key
+    ca:
```

Comment:
We had quite a discussion about this in the `nats` chart, and ended up making the decision to explicitly separate secret/config map for the Client Cert and the CA Certs

Reason being is that a single certificate that includes it's CA is normally the CA that signed the Client Cert - not the CA list that should be used to trust the Server Cert.  Of course often times they are the same but not always.

So maybe we need to duplicate this, and separate it into `tlsClient` and `tlsCA`

Here is what we ended up doing for `tlsCA` in the NATS chart - it accepts either a Secret Name or a Config Map name: https://github.com/nats-io/k8s/blob/88744c4b5371fb71c547b8e7efd0767ec485b405/helm/charts/nats/values.yaml#L32